connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorCheckpointConfigTest.java

```diff
@@ -95,4 +96,29 @@ public void testConsumerConfigsForOffsetSyncsTopic() {
                 "source1->target2|ConnectorName|" + MirrorCheckpointConfig.OFFSET_SYNCS_TARGET_CONSUMER_ROLE,
                 offsetSyncsTopicTargetConsumerConfig.get("client.id"));
     }
+
+    @Test
+    public void testValidate() {
```

Comment:
Can we also add a case to make sure that if the connector is disabled, we don't report errors on otherwise-invalid configs?

Reply:
I added more tests